workspaces/spectacle/src/index.ts

```diff
@@ -144,23 +144,14 @@ export interface IOpticContext {
 
 ////////////////////////////////////////////////////////////////////////////////
 
-export type AsyncStatus<T> =
```

Comment:
Moved out into `ui-v2/src/types` since spectacle doesn't actually contain async state (it just provides a promise that has data or an error)